Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -260,6 +258,9 @@ - (void)configureViews
     CGSize bodyMediaContentSize = [self bodyMediaSize];
     CGSize bodyTextContentSize = [self bodyTextSizeWithIncludeMargins:NO];
 
+    [self.bubbleView addSubview:self.stackView];
```

Comment:
Why this change - as far as I can tell we never remove the bubbleView.?

Reply:
In `prepareForReuse` we remove all subviews from `bubbleView`.

Reply:
oh sorry I somehow misread this as `[self addSubview:bubleView]`

This looks fine!